inbm/dispatcher-agent/dispatcher/sota/sota.py

```diff
@@ -372,6 +372,9 @@ def execute_from_manifest(self,
                 self._dispatcher_broker.send_result(SOTA_FAILURE)
                 raise SotaError(SOTA_FAILURE)
 
+    def _is_reboot_device(self) -> bool:
+        return self._reboot_device not in ["No", "N", "n", "no", "NO"]
+
```

Comment:
**<h2>$\color{#15F4EE}Miscellaneous$</h2>**
**Suggestion:** Implement the _is_reboot_device method to ensure it correctly checks the reboot condition based on the updated variable name _reboot_device.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
    def _is_reboot_device(self) -> bool:
        return self._reboot_device.lower() not in ["no", "n"]
```